monai/deploy/operators/dicom_series_to_volume_operator.py

```diff
@@ -112,7 +112,7 @@ def generate_voxel_data(self, series):
         # with the NumPy array returned from the ITK GetArrayViewFromImage on the image
         # loaded from the same DICOM series.
         vol_data = np.stack([s.get_pixel_array() for s in slices], axis=0)
-        vol_data = vol_data.astype(np.int16)
+        vol_data = vol_data.astype(np.uint16)
```

Comment:
This can only be safe done when the pixel data is in fact unsigned, and the following DICOM tag tells the fact (when value == 1)

Tag	(0028,0103)
Type	Required (1)
Keyword	PixelRepresentation
Value Multiplicity	1
Value Representation	Unsigned Short (US)

Example Values	
  1
  0

Enumerated Values:
0000H
    unsigned integer.
0001H
    2's complement